### DISCUSSION TITLE
Termux on older Androids

### DISCUSSION THREAD

**Discussion by sage-nmm** [termux/termux-app#3890] [Q&A] [answered]
From what i've seen, termux "only" runs on Android 7+.
I've searched multiple releases but only recently are .apks provided and i didn't find any release notes mentioning version changes.
Any chances i could find an apk that supports older Android (6 or 4)? I'm trying to give ewaste some use.

**Comment by agnostic-apollo** [comment, marked as answer]
Read https://github.com/termux/termux-app#Installation

**Comment by paraman2020svg-prog** [comment]
Hey there, can I join in

**Comment by paraman2020svg-prog** [comment]
Yes , I also give Huawei Y5 2017 MYA-L22 some use , I workings on termux 0.73 offline bootstrap version
